src/main/java/io/saagie/maven/CreateJobMojo.java

```diff
@@ -19,9 +19,31 @@ public void execute() {
             managerRestClient.checkManagerConnection();
             String directory = project.getBuild().getDirectory();
             String filename = managerRestClient.uploadFile(directory, managerProperties.getJarName());
-            String body = "{\"platform_id\":\"" + managerProperties.getPlatformId() + "\",\"capsule_code\":\"" + managerProperties.getJobType() + "\",\"category\":\"" + managerProperties.getJobCategory() + "\",\"current\":{\"template\":\"java -jar {file} "+managerProperties.getArguments()+"\",\"file\":\"" + filename + "\"},\"options\":{\"language_version\":\"" + managerProperties.getLanguageVersion() + "\"},\"cpu\":" + managerProperties.getCpu() + ",\"memory\":" + managerProperties.getMem() + ",\"disk\":" + managerProperties.getDisk() + ",\"manual\":true,\"name\":\"" + managerProperties.getJobName() + "\",\"retry\":\"\",\"schedule\":\"R0/2016-07-06T15:47:52.051Z/P0Y0M1DT0H0M0S\"}";
+            String body = "{" +
```

Comment:
Maybe it will be easier to maintain this body by creating a Builder.

Reply:
done